### PR TITLE
Documentation enhanced concerning issue #888

### PR DESCRIPTION
**Pull request by teroneko**
Documentation enhanced concerning issue #888 (https://github.com/GitTools/GitVersion/issues/888)

### CODE REVIEW

File: docs/input/docs/configuration.md [hunk 1]
```diff
@@ -137,6 +137,16 @@ When using `mode: ContinuousDeployment`, the value specified in
 `continuous-delivery-fallback-tag` will be used as the pre-release tag for
 branches which do not have one specified. Default set to `ci`.
 
+Just to clarify: For a build name without "...-ci-\<buildnumber>" or in other
+words without a PreReleaseTag (ergo "PreReleaseTag":"" when typing GitVersion)
```
Comment (asbjornu) [inline docs/input/docs/configuration.md]:
```suggestion
words without a `PreReleaseTag` (ergo `"PreReleaseTag":""` in GitVersion's JSON output)
```

File: docs/input/docs/configuration.md [hunk 2]
```diff
@@ -137,6 +137,16 @@ When using `mode: ContinuousDeployment`, the value specified in
 `continuous-delivery-fallback-tag` will be used as the pre-release tag for
 branches which do not have one specified. Default set to `ci`.
 
+Just to clarify: For a build name without "...-ci-\<buildnumber>" or in other
```
Comment (asbjornu) [inline docs/input/docs/configuration.md]: Is the backslash in `...-ci-\<buildnumber>` intentional?

```suggestion
Just to clarify: For a build name without `...-ci-\<buildnumber>` or in other
```

File: docs/input/docs/configuration.md [hunk 3]
```diff
@@ -137,6 +137,16 @@ When using `mode: ContinuousDeployment`, the value specified in
...
+words without a PreReleaseTag (ergo "PreReleaseTag":"" when typing GitVersion)
+at the end you would need to set continuous-delivery-fallback-tag to '':
```
Comment (asbjornu) [inline docs/input/docs/configuration.md]:
```suggestion
at the end you would need to set `continuous-delivery-fallback-tag` to an empty string (`''`):

```

File: docs/input/docs/configuration.md [hunk 4]
```diff
@@ -137,6 +137,16 @@ When using `mode: ContinuousDeployment`, the value specified in
...
+```
```
Comment (asbjornu) [inline docs/input/docs/configuration.md]:
```suggestion
```yaml
```

File: docs/input/docs/configuration.md [hunk 5]
```diff
@@ -137,6 +137,16 @@ When using `mode: ContinuousDeployment`, the value specified in
...
+mode: ContinuousDeployment
+continuous-delivery-fallback-tag: ''
+...
+```
+Doing so can be applicable if you use your master branch as release branch.
```
Comment (asbjornu) [inline docs/input/docs/configuration.md]:
```suggestion

Doing so can be helpful if you use your `master` branch as a `release` branch.
```

File: docs/input/docs/configuration.md (hunk 2 above)
Comment (teroneko) [reply docs/input/docs/configuration.md]: Now it is obsolete. It is needed in favor of escape the angle bracket. But when surrounded by backticks to represent code it is not needed anymore.

**Comment by teroneko** [comment]
Men, I've never edited a pull reqeust in GitHub. Completely new territorium. Sorry for the many new commits... I see you can batch changes. Very confusing when not worked with before.

**Comment by gep13** [comment]
>@teroneko said..
Men, I've never edited a pull reqeust in GitHub. Completely new territorium. Sorry for the many new commits... I see you can batch changes. Very confusing when not worked with before.

I took the liberty of rebasing your changes into the two main commits.  Let me know if this causes you problems on your local working copy of the repository, and we can get that sorted for you.

File: docs/input/docs/configuration.md (hunk 2 above)
Comment (asbjornu) [reply docs/input/docs/configuration.md]: So should my suggested change be commited, then?

Comment (teroneko) [reply docs/input/docs/configuration.md]: Yea, but I never authored pull request changes, I accidentally created a bunch of unnecessary commits. Sorry. I am not sure. I want to take your change, but then I end with a fresh commit. But their is one change left off, as the escape is not needed anymore. But then I would create another commit to remove that single backslash. So all these commits would be visible when this pull request is merged into develop/master, right? What so ever, I manually added the line into his own commit.

**Comment by asbjornu** [comment]
Thank you for your contributions, @teroneko!

**Pull request merged**